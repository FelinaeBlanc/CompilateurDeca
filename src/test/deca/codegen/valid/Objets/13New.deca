class A {
    int x = 0;
    int y = 13;

    int getX(int a){
        return a*x;
    }

    float getY(){
        int z = this.y;
        return (float)(z + this.getX(1));
    }
}

{
    A a = new A() ;
    println(a.getY());
}
