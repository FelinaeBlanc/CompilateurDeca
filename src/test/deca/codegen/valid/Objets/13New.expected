1.30000e+01
